x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/datafeed/DatafeedJob.java

```diff
@@ -350,7 +353,7 @@ private void run(long start, long end, FlushJobAction.Request flushRequest) thro
                     if (isIsolated) {
                         return;
                     }
-                    LOGGER.debug("[" + jobId + "] error while posting data", e);
+                LOGGER.error(new ParameterizedMessage("[{}] error while posting data", jobId), e);
```

Comment:
audit, too?

Reply:
`ProblemTracker` makes sure this is audited